### DISCUSSION TITLE
Join the Iroh Discord!

### DISCUSSION THREAD

**Discussion by b5** [n0-computer/iroh#1856] [Announcements]
We have a discord! Join it!

https://iroh.computer/discord

**Comment by maboesanman** [comment]
Looks like the invite has expired. Can you issue a new invite with no expiration?

**Reply by divagant-martian** [reply]
here you go! https://discord.gg/QNsS5dEgqw

**Reply by b5** [reply]
🤦 thank you so much for asking. I've just updated the site with a non-expiring link. https://iroh.computer/discord should be fixed in a moment: https://github.com/n0-computer/iroh.computer/pull/81

**Comment by maacl** [comment]
Invite states it is invalid.

**Reply by dignifiedquire** [reply]
I just tested https://iroh.computer/discord and it seems to work fine

**Comment by maacl** [comment]
That is super odd. I just clicked the link and got invaild invite. Now it
works.
